pkg/liqoctl/install/handler.go

```diff
@@ -33,23 +33,35 @@ import (
 // HandleInstallCommand implements the "install" command. It detects which provider has to be used, generates the chart
 // with provider-specific values. Finally, it performs the installation on the target cluster.
 func HandleInstallCommand(ctx context.Context, cmd *cobra.Command, baseCommand, providerName string) error {
-	if !klog.V(4).Enabled() {
-		klog.SetLogFilter(logsutils.LogFilter{})
-	}
+	verbose, err := cmd.Flags().GetBool("verbose")
+	utilruntime.Must(err)
+
+	printer := common.NewPrinter("", common.Cluster1Color)
+	logsutils.SetupLogger(printer, verbose)
```

Comment:
Could it make sense to move this part to the root, to avoid having to call it for all commands?